Valigator.Core/Valigator.Core.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>This package contains fluent validators for data.</Description>
     <PackageTags>validation</PackageTags>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>2.0.1</Version>
```

Comment:
Do I need to update version anywhere else to update the version of the other packages?